content/en/docs/Examples/boot_assessment_trusted_boot.md

```diff
@@ -0,0 +1,118 @@
+---
+title: "Enabling Automatic Boot Assessment with Trusted Boot"
+linkTitle: "Enabling Automatic Boot Assessment with Trusted Boot"
+weight: 4
+description: This section describe examples on how to enable Automatic Boot Assessment with Trusted Boot in your own services.
+---
+
+
+# Enabling Automatic Boot Assessment with Trusted Boot
+
+In this tutorial, we will walk through how to configure Kairos to enable **automatic boot assessment**, where the boot loader can determine the health of a boot entry. Specifically, we'll configure systemd services to trigger the `boot-complete.target` to mark boot entries as *good* or *bad*. We'll also cover how to implement automatic reboots when a service fails, allowing retries of a boot entry until success or exhaustion of attempts.
```

Comment:
By reading it entirely something is not really clear to me: here we say that the example is on how to enable the automatic boot assessment, but later we talk about marking an arbitrary success or failure impacts the boot assessment - I think we should be explicit here in answering the following questions: 

- what are system defaults ? is boot assessment enabled?
- is Kairos having a default boot assessment strategy?
-  if Kairos does have a default , how it behaves, what are the limitations? 
-  If it doesn't, why we don't have a default for it?
-  finally, how can we extend the boot assessment by including our own custom services to impact the system state check?

Reply:
we currently dont have any assesment strategy...yet.

The idea was to write the docs first and then start enabling it ourselves, plus adding e2e tests for it.

Current boot assessment is set as the default, i.e. if multi-user.target is reached, then it considers the system as good and marks it as such.

But I get what you mean, I will redo this to extend to introduce first the current states of assesment on kairos and then keep this as how can be extended to support more things.

Reply:
Expanded!